src/components/ProjectView/ImageCarousel.vue

```diff
@@ -1,47 +1,28 @@
 
 <script setup>
-import { ref, defineProps } from 'vue'
+import { ref } from 'vue'
 import { useStore } from 'vuex'
 import { Carousel, Slide  } from 'vue3-carousel'
 import 'vue3-carousel/dist/carousel.css'
-
+import ImageAnalyzer from '../ImageAnalyzer.vue'
 
 const store = useStore()
 const currentSlide = ref(0)
-const props = defineProps(['data'])
-let smallImageBasename = ref('')
-let largeImageSrc = ref('')
-
-let data = ref(props.data)
-
-// Checking if image isSelected to either add or remove yellow borderline
-const isSelected = (item) => store.getters.isSelected(item)
-
-// Getting large images by using the thumbnail's basename and finding the large image that matches it
-const getLargeImageSource = () => {
```

Comment:
this method became a getter in the store since that is the source of truth for our images, at least for the project view